### PR TITLE
Issue#628

### PR DESCRIPTION
**Pull request by C-Lion**
Before:
issue628-before

After:
issue628completed

### CODE REVIEW

File: app/views/pages/mentorship.html.erb [hunk 1]
```diff
@@ -5,7 +5,9 @@
...
+        <p>The program is designed to support your pursuit of becoming a software developer. Additionally, we have a thriving <a href="https://operation-code.slack.com/messages/free-code-camp/">#free-code-camp</a> study group, and location-based communities, from <a href="https://operation-code.slack.com/messages/seattle/">#seattle</a> to <a href="https://operation-code.slack.com/messages/washingtondc/">#washingtondc</a> to meet others locally in your area and organize <a href="https://www.meetup.com/pro/operationcode/">Operation Code meetups</a>.</p>
+        <%= link_to 'Join now &raquo;'.html_safe, new_veteran_registration_path, class: 'btn btn-large' %>
```
Comment (rickr) [inline app/views/pages/mentorship.html.erb]: Can we center this button?

**Comment by C-Lion** [comment]
Sure can! On it now.

**Comment by C-Lion** [comment]
pr641-centered

**Comment by C-Lion** [comment]
Done: 
pr641-centered

**Pull request merged**